.github/ISSUE_TEMPLATE/UP Onboarding-Receiver settings.md

```diff
@@ -0,0 +1,57 @@
+name: Receiver UP Onboarding Receiver Settings Template
```

Comment:
The formating of this block does not appear to work.  Maybe missing opening "---"?